core/controllers/suggestion.py

```diff
@@ -50,6 +50,38 @@ def _require_valid_suggestion_and_target_types(target_type, suggestion_type):
             'Invalid suggestion_type: %s' % suggestion_type)
 
 
+def _render_suggestions(self, target_type, suggestions):
```

Comment:
can we do this without passing the self?

You can either create a parent class that will have this method or create a separate helper method that will return the required dict.

Reply:
I went with the parent class option. Thanks.